packages/connect-query/src/use-infinite-query.ts

```diff
@@ -35,9 +35,6 @@ import type { DisableQuery } from "./utils.js";
 
 /**
  * Query the method provided. Maps to useInfiniteQuery on tanstack/react-query
- *
- * @param methodSig
- * @returns
```

Comment:
Quite distracting warnings in JetBrains IDEs:

<img width="616" alt="w" src="https://github.com/user-attachments/assets/f4963e19-4d6d-4cba-9eaf-ddadf360f2e8">

Reply:
Ick. Yeah let's nuke em